### PR TITLE
feat: allow render to return string

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: src/server/render.tsx [hunk 1]
```diff
@@ -310,6 +310,7 @@ export async function suspenseRender(
     function render() {
       try {
         body = renderToString(vnode);
+        return body;
```
Comment (lucacasonato) [inline src/server/render.tsx:322]: In the "catch a promise path" this should return an empty string.

**Review by lucacasonato — COMMENTED** [review]
Missing tests

File: src/server/render.tsx (hunk 1 above)
Comment (crowlKats) [reply src/server/render.tsx:322]: should i change the typings to RendererModule.render to return only string and not null then?

Comment (lucacasonato) [reply src/server/render.tsx:322]: Yes

**Review by lucacasonato — APPROVED** [review]
LGTM

**Pull request merged**